ncm-metaconfig/src/main/metaconfig/nrpe/pan/schema.pan

```diff
@@ -0,0 +1,27 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+declaration template metaconfig/nrpe/schema;
+
+include 'pan/types';
+
+@{Configuration options for Nagios Remote Plugin Executor (NRPE)}
+type nrpe_service = {
+    'log_facility'              : string = 'daemon'
+    'pid_file'                  : string = '/var/run/nrpe.pid'
+    'server_port'               : type_port = 5666
+    'server_address'            ? string
+    'nrpe_user'                 : string = 'nagios'
+    'nrpe_group'                : string = 'nagios'
+    'allowed_hosts'             : type_hostname[]
+    'dont_blame_nrpe'           : boolean = false
+    'command_prefix'            ? string
+    'debug'                     : boolean = false
+    'command_timeout'           : long = 60
+    'connection_timeout'        : long = 300
```

Comment:
Comments in `nrpe.cfg` say "Do not set this too low." I wonder what that means :grin: